app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+  def updates_comment_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
**Optional**
- It'd be awesome if you make use of the rails helper method after_save to directly recalled to the record (Comment, Post, etc) models like so `after_save :update_comments_counter`